prow/config/postsubmits.yaml

```diff
@@ -95,7 +95,7 @@ postsubmits:
     path_alias: maistra.io/ior
     skip_report: false
     branches:
-      - maistra-1.2
+      - maistra-2.0
```

Comment:
ior doesn't have a 2.0 branch. You can leave just 1.1 here. 1.2 existed here because it was my playground when developing automator.